package.json

```diff
@@ -53,24 +53,24 @@
     "karma-sinon-chai": "^1.2.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "1.7.0",
-    "less": "^2.6.1",
+    "less": "^2.7.1",
     "less-loader": "^2.2.3",
     "mocha": "^2.3.4",
-    "mocha-loader": "^0.7.1",
     "null-loader": "^0.1.1",
     "phantomjs-prebuilt": "^2.1.7",
-    "script-loader": "^0.6.1",
+    "script-loader": "^0.7.0",
     "style-loader": "^0.13.1",
     "vue-hot-reload-api": "^1.3.2",
-    "vue-html-loader": "^1.2.2",
-    "vue-loader": "8.2.2",
+    "vue-html-loader": "^1.2.3",
+    "vue-loader": "8.5.3",
     "watai": "0.7.0",
-    "webpack": "1.12.14"
+    "webpack": "1.13.1"
   },
   "dependencies": {
     "Chart.StackedBar.js": "regaddi/Chart.StackedBar.js#1.0.3",
     "admin-lte": "^2.3.5",
     "babel-polyfill": "^6.9.1",
+    "babel-runtime": "^6.9.1",
```

Comment:
Just because npm 2.x bundled with node 4 can't handle peerDepedencies properly :disappointed: 
The dependency is downloaded for npm to be happy but not used.